cmd/gaia/simulation/sim_test.go

```diff
@@ -0,0 +1,21 @@
+package simulation
```

Comment:
++ on this as a new package

Reply:
unfortunately, turns out we need to access keepers, which are unexported

Reply:
hmm - clarify the problem?